polaris.shopify.com/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM node:16
+
+ENV PORT 3000
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Installing dependencies
+COPY package*.json /usr/src/app/
```

```suggestion
COPY package*.json .
```

The path to application directory is already set by the above [`WORKDIR` instruction](https://docs.docker.com/engine/reference/builder/#workdir)